docs/README.md

```diff
@@ -1,5 +1,11 @@
 #CAPx Documentation
 
+##Web server system requirements
+
+- Linux xxx
```

Comment:
Thanks for this Kevin,

This module does not necessarily require Linux. Drupal.org documents the webserver requirements here: https://www.drupal.org/docs/7/system-requirements/web-server. We could link to that page as a starting point maybe? 

This module has additional requirements because it uses a third party library called Guzzle. It currently uses version 6. Perhaps we could also link or copy the requirements here for Guzzle: http://docs.guzzlephp.org/en/latest/overview.html#requirements

Thanks for this contribution!